.github/workflows/test-webservice.yml

```diff
@@ -35,7 +35,7 @@ jobs:
     - uses: actions/checkout@v2
     - uses: maxim-lobanov/setup-xcode@v1.2.3
       with:
-        xcode-version: latest
+        xcode-version: '13.0'
```

Comment:
Looking at the output of the workflow, the GitHub runners currently ship with Xcode build 13A5192i which should be beta 3 if I'm not wrong.
So the toolchain itself is up to date, but it probably fails at the OS version, as there seemingly aren't any macOS 12 runners ://

Reply:
Yeah, that's correct. There are no macOS 12 runners yet, but Apodini is enlisted to be among the first testers once they are launched. I put the macOS version number to 12 for two reasons:
* We'd notice if they launch macOS 12 runners before we get our own runners working.
* When I put it to 11 with Xcode 13, the CI even manages to compile the whole project (even though the macOS(.v12) SPM tag clearly states that macOS 12 is required). It then fails at startup because the `_Concurrency` library is missing.  That's just confusing. As is we clearly see the runners are not fit for running the project.